### PR TITLE
refactor flash messages in case of using Manual Engine

### PR DESCRIPTION
**Pull request by naumvd95**
Manual engine doesnot support scaling and deleting

### CODE REVIEW

File: kqueen_ui/blueprints/ui/views.py [hunk 1]
```diff
@@ -618,6 +623,11 @@ class ClusterResize(KQueenView):
 
     def handle(self, cluster_id):
         cluster = self.kqueen_request('cluster', 'get', fnargs=(cluster_id,))
+
```
Comment (atengler) [inline kqueen_ui/blueprints/ui/views.py]: Remove this part pls, I have already PR prepared for improving the message overall and I don't want any conditions checking for specific engine in a code which can by written generically just fine.

File: kqueen_ui/blueprints/ui/views.py [hunk 2]
```diff
@@ -555,12 +555,17 @@ class ClusterDelete(KQueenView):
 
     def handle(self, cluster_id):
         cluster = self.kqueen_request('cluster', 'get', fnargs=(cluster_id,))
+        msg = 'Cluster {} successfully deleted.'.format(cluster['name'])
+
```
Comment (atengler) [inline kqueen_ui/blueprints/ui/views.py]: In this case there is really not a way around explicitly testing if we are on Manual engine, so I guess this is OK

File: kqueen_ui/blueprints/ui/views.py (hunk 1 above)
Comment (atengler) [reply kqueen_ui/blueprints/ui/views.py]: This is the PR: https://github.com/Mirantis/kqueen-ui/pull/28 - it handles both Jenkins and Manual engines without explicit conditions

Comment (naumvd95) [reply kqueen_ui/blueprints/ui/views.py]: okok

File: kqueen_ui/blueprints/ui/views.py (hunk 2 above)
Comment (naumvd95) [reply kqueen_ui/blueprints/ui/views.py]: done

**Comment by naumvd95** [comment]
pls recheck @atengler

**Pull request merged**